aioftp/server.py

```diff
@@ -35,6 +36,7 @@
     "AbstractServer",
     "Server",
 )
+IS_PY37 = sys.version_info[:2] == (3, 7)
```

Comment:
And maybe use `>=` instead of `==`?

Reply:
I mean rename this as something like `IS_PY37_PLUS`

Reply:
Done!